courseme/util/json.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""JSON utility functions"""
+
+from __future__ import absolute_import
+import json as j
+
+from datetime import datetime
+
+class _CustomEncoder(j.JSONEncoder):
```

Comment:
Note the leading underscore on the name.  It's good practice to communicate what's public and what's not when writing a module.  It's not enforced by Python, but a leading underscore tells me, the consumer of this module, that I can't rely on `_CustomerEncoder` always being there or always having the same behaviour.  And it frees you up, as the authour or a module, to change it as you need without worrying about backward compatibility.